interface-definitions/system-logs.xml.in

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="system">
+    <children>
+      <node name="logs" owner="${vyos_conf_scripts_dir}/system-logs.py">
```

Comment:
Can we name this `system logging` instead?

Reply:
I have no predisposition for any of the name variants in this case - this is only the question of taste. Would you like to change the name?

Reply:
As for me, it logically should be in `set system syslog` section
As we have such description:
```
set system 
...
 > syslog       System logging
...
```
It's just my opinion. @andamasov doesn't like this idea

Reply:
@sever-sever, I see at least two different components that are not related anyhow to Syslog and should be controlled by this:
- logrotate
- systemd-journald
That is why it was placed in an independent node.

Reply:
I'd also not place it under syslog node as this is logrotate and not syslog ;)

Reply:
I prefer to set all log options  under one "logical node", which will include all other logs/logging/syslog
I understand that it 2 different functions
But it looks strange
```
set system logging
set system logs
set system syslog (description System logging)
```

Reply:
For 1.4 we could also migrate syslog to a child of „system logging“